### PR TITLE
fix: update functions deploy script

### PR DESCRIPTION
**Pull request by tphokas**
- the script would fail on first run because the `build` directory would not be created
- on failure the script would not exit
- add execute permissions on deploy.sh
```
starting npx build for webrequest

./deploy.sh: line 28: build/index.js: No such file or directory
./deploy.sh: line 29: cd: build: No such file or directory

built package for webrequest, ready to deploy
```

### CODE REVIEW

File: stedi-cloud/functions/web-request/deploy.sh [hunk 1]
```diff
@@ -19,8 +21,8 @@ buildfunction() {
     if test -d "./build/"
```
Comment (ant1eicher) [inline stedi-cloud/functions/web-request/deploy.sh:21]: Why do we even need the test? `rm -rf` will not exit non-zero if `build` doesn't exist.

**Review by ant1eicher — APPROVED** [review]
🚢
